Jenkinsfile-remote-testing

```diff
@@ -0,0 +1,152 @@
+#!groovy
+@Library('waluigi@release/7') _
+
+properties([
+  parameters([
+    string(name: 'name', defaultValue: 'bedorck-run', desciption: 'Name of the testing run', trim: true),
+    string(name: 'browser', defaultValue: 'chrome', desciption: 'Testing browser', trim: true),
+    string(name: 'provider', defaultValue: 'lambdatest', desciption: 'Remote testing provider', trim: true),
+    string(name: 'bucket', defaultValue: '1', desciption: 'Current testing bucket', trim: true),
+    string(name: 'buckets', defaultValue: '1', desciption: 'Testing buckets', trim: true),
+    string(name: 'branch', defaultValue: 'develop', desciption: 'Testing branch', trim: true)
+  ])
+])
+
+// bedrock-auto test on TinyMCE for given browser/provider
+def testRemote(browser, provider) {
+  BUCKETS = 1
+  dir('tinymce') {
+    pwd
+    // Test other browsers even if one fails.
+    catchError(stageResult: 'FAILURE') {
+      remoteCredentials(provider) {
+        for (i = 1; i <= BUCKETS; i++) { // 1-indexed
+          sh 'yarn browser-test --bedrock-browser=' + browser + ' --remote=' + provider +
+            ' --buckets=15 --bucket=' + i + ' --chunk=50'
```

Comment:
the `--buckets=15` is supposed to be `--buckets=BUCKETS` and I had temporarily changed it for testing purposes and forgot to put it back

Reply:
Actually, this whole function is not used in this case so it can be removed.